modules/node_modules/@ncigdc/components/Aggregations/ExactMatchFacet.js

```diff
@@ -82,9 +82,10 @@ const ExactMatchFacet = compose(
                     ], false),
                   }}
                   dark={!!inputValue}
-                  onClick={() => setInputValue('')}
```

Comment:
Links should already be doing that unless we forgot to pass props along somewhere